tests/scene/test_code_edit.h

```diff
@@ -3476,45 +3476,45 @@ TEST_CASE("[SceneTree][CodeEdit] completion") {
 		code_edit->confirm_code_completion();
 		CHECK(code_edit->get_line(0) == "stest");
 
-		/* Favorize closer location. */
+		/* Favorize good capitalization. */
```

Comment:
While we're here, could we replace "Favorize" with "Prefer", or "Prioritize" at least?

Reply:
Also "over" instead of "to"